src/main/java/Task.java

```diff
@@ -0,0 +1,49 @@
+public class Task {
```

Comment:
You can try making this Task class as an abstract class!

Reply:
Okay noted! Will read up and improve my code!